javatests/arcs/android/storage/database/DatabaseImplTest.kt

```diff
@@ -664,6 +664,41 @@ class DatabaseImplTest {
         assertThat(entityOut).isEqualTo(parentEntity)
     }
 
+    @Test
```

Comment:
It'd be good to add a test to ensure the migration works as expected as well.

Reply:
Added some in DatabaseUpgradeTest. Let me know if you were thinking something different.